protos/runner_services_pb.rb

```diff
@@ -0,0 +1,21 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# Source: runner.proto for package 'fastlane_c_i'
+
+require "grpc"
+require "runner_pb"
+
+module FastlaneCI
+  module Runner
+    class Service
+      include GRPC::GenericService
+
+      self.marshal_class_method = :encode
+      self.unmarshal_class_method = :decode
+      self.service_name = "fastlane_c_i.Runner"
```

Comment:
Maybe rename to `fastlane_ci` 👍

Reply:
So the package name of `fastlane_c_i` comes from the proto. I chose that particular string because it gets generated to `module FastlaneCI` in the ruby source. `fastlane_c_i` is only visible in the proto definition. I don't have a particular preference either way. I am open to changing it to anything, but I do want to avoid a namespace `FastlaneCi` existing in addition to `FastlaneCI`